src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2/console
+spring.datasource.url=jdbc:h2:C:/Users/vasila1/IdeaProjects/shop;schema="shop"
```

Comment:
this should be a relative path to the project. Anyone should be able to run this code on their machine without further tweaks and not everyone has the folder vasila1 in Users